src/lib.rs

```diff
@@ -21,6 +23,7 @@
 //!
 //! - x86_64
 //! - arm64 (experimental)
```

Comment:
I think elsewhere you mark arm64 as non-experimental.

Reply:
Centainly, that would be in the `prepare-release` PR after this is merged :)